source/jquery.flot.js

```diff
@@ -984,6 +987,25 @@ Licensed under the MIT license.
             executeHooks(hooks.bindEvents, [eventHolder]);
         }
 
+        function addEventHandler(event, handler, eventHolder, priority) {
+            var key = eventHolder + event;
+            var eventlist = eventManager[key];
+            if (eventlist === undefined) {
+                eventlist = [];
+            }
+
+            // var newEventList = [];
+            eventlist.push({"event": event, "handler": handler, "eventHolder": eventHolder, "priority": priority});
+            eventlist.sort(function(a, b) { return b.priority - a.priority });
+            for (var i = 0; i < eventlist.length; i++) {
```

Comment:
It doesn’t look like you use the index for anything — couldn’t this be done using array’s `forEach` method?  E.g.:
```
eventlist.forEach( eventData => {
  eventData.eventHolder.unbind(eventData.event, eventData.handler);
  eventData.eventHolder.bind(eventData.event, eventData.handler);
} );
```

Reply:
Yep.